packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from 'terra-section-header';
+import Toggle from 'terra-toggle';
+
+const VALID_LEVELS = [1, 2, 3, 4, 5, 6];
+
+const propTypes = {
+  /**
+   * Content in the body of the toggle section header component that will be expanded or hidden.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text to be displayed in the section header.
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, or 6.
+   */
+  level: PropTypes.oneOf(VALID_LEVELS),
+  /**
+   * Callback function triggered when toggle-section-header is closed.
+   */
+  onClose: PropTypes.func,
+  /**
+   * Callback function triggered when toggle-section-header is opened.
+   */
+  onOpen: PropTypes.func,
+  /**
+   * Used to set props and HTML attributes on the toggle-section-header section-header.
```

Comment:
This document seems a little awkward because of the double section-header. I would just leave off the toggle-section-header.

Reply:
Fixed ab0223ba789915df78011e55da8e4093f191e53c